.github/workflows/ci.yml

```diff
@@ -33,6 +33,34 @@ jobs:
     outputs:
       check_if_build: ${{ steps.check.outputs.check_if_build }}
 
+  build_lambda:
```

Comment:
what's the purpose of this job? seems like we are building that package again on release anyway.

It's still could be useful for pr testing purposes, right? In that way we can call it `test_aws_build` and call it only for pr. And without `--release` flag so we don't spend much time on build that is thrown away

Reply:
Fixed in 0256bfcbb3acffd6ed9f8ebf4e65a59fa91af274

Reply:
Yes, it's for testing purposes.